lib/runtime/index.js

```diff
@@ -165,9 +165,9 @@ function watch(exported, setters, key, namespaces) {
 
 Rp.watch = watch;
 
-function setESModuleAndOwner(module) {
-  utils.setESModule(module.exports);
-  Entry.getOrCreate(module.exports, module);
```

Comment:
This helps prevent confusion since we know by looking at it that it's not the real real `module`.